src/jit/codegencommon.cpp

```diff
@@ -10582,6 +10595,16 @@ void CodeGen::genPreserveCalleeSavedFltRegs(unsigned lclFrameSize)
         regMaskTP regBit = genRegMask(reg);
         if ((regBit & regMask) != 0)
         {
+#ifdef FEATURE_AVX_SUPPORT
+            // Vzeroupper needs to be issued before copyIns (vmovupd etc.) that are AVX instructions
+            if (compiler->getFloatingPointInstructionSet() == InstructionSet_AVX &&
+                !bVzeroupperIssued &&
+				getEmitter()->IsAVXInstruction(copyIns))
+            {
+                instGen(INS_vzeroupper);
+                bVzeroupperIssued = true;
+            }
+#endif
```

Comment:
I'm wondering why we would have the case where we have no AVX instructions emitted (i.e. we didn't issue the vzeroupper above), but we have an AVX copy instruction. That is, if we are using AVX encodings and have no AVX instructions issued, then how can we be using callee save floating point registers?